packages/amplify-category-function/package.json

```diff
@@ -33,7 +33,7 @@
     "folder-hash": "^4.0.2",
     "fs-extra": "^8.1.0",
     "globby": "^11.0.3",
-    "graphql-transformer-core": "^7.6.1",
```

Comment:
question: Is this change required? wouldn't ^7.6.1 pull in ^7.6.2 automatically on publish?

Reply:
Technically, yes. But we generate this PR using a script which updates the version explicitly to keep it consistent across the repository.

Reply:
Thanks